src/user_config.yaml

```diff
@@ -8,26 +8,25 @@ global:
   postcode_csv_check: True
   load_ni_data: False
   # Freezing settings
-  run_with_snapshot_until_freezing: True      # Run until point of freezing - last run is first freeze
```

Comment:
Are we able to keep develop on the 'normal' run until we've talked through the new setup with everyone?  We could do that at tomorrow's scrum, and then merge this as is?

Reply:
Sure! Good shout.